vrp-pragmatic/src/lib.rs

```diff
@@ -46,9 +47,16 @@ pub fn get_unique_locations(problem: &Problem) -> Vec<Location> {
     CoordIndex::new(problem).unique()
 }
 
-fn format_time(time: Float) -> String {
+// vrp_core's Timestamps are f64s that could go far beyond what unix timestamps support
+const MIN_TIMESTAMP: i64 = OffsetDateTime::new_in_offset(Date::MIN, Time::MIDNIGHT, UtcOffset::UTC).unix_timestamp();
+const MAX_TIMESTAMP: i64 = OffsetDateTime::new_in_offset(Date::MAX, Time::MAX, UtcOffset::UTC).unix_timestamp();
+
+fn format_time(time: Timestamp) -> String {
+    let time: i64 = (time as i64).clamp(MIN_TIMESTAMP, MAX_TIMESTAMP);
```

Comment:
haven't seen anything near `MIN_TIMESTAMP` but the handling is the same and it reduces the window of failure